LibreHardwareMonitorLib/Hardware/Cpu/Amd17Cpu.cs

```diff
@@ -236,17 +237,25 @@ public void UpdateSensors()
                 //Core
                 if ((smuSvi0Tfn & 0x01) == 0)
                 {
-                    svi0PlaneXVddCor = (smuSvi0TelPlane0 >> 16) & 0xff;
-                    vcc = 1.550 - vidStep * svi0PlaneXVddCor;
+                    if (cpu.Model == 0x71)
+                        svi0PlaneXVddCor = (smuSvi0TelPlane1 >> 16) & 0xff;
```

Comment:
What about (same as below):
`svi0PlaneXVddCor = ((cpu.Model == 0x71 ? smuSvi0TelPlane1 : smuSvi0TelPlane0) >> 16) & 0xff;`

Reply:
i just thought this looks cleaner but sure